jmxtrans-output/jmxtrans-output-influxdb/src/test/java/com/googlecode/jmxtrans/model/output/InfluxDbWriterTests.java

```diff
@@ -74,11 +72,13 @@
 
 	private static final String DATABASE_NAME = "database";
 	private static final String HOST = "host.example.net";
+	private Map<String,String> TAGS = new HashMap<>();
```

Comment:
`TAGS` should not be a class member (and ALL CAPS variable names indicate a constant) but should be moved to each test method.